Packs/Cyberpion/pack_metadata.json

```diff
@@ -17,5 +17,14 @@
     "marketplaces": [
         "xsoar",
         "marketplacev2"
+    ],
+    "dependencies": {
+        "CoreAlertFields": {
+            "mandatory": true,
+            "display_name": "Core Alert Fields"
+        }
+    },
+    "displayedImages": [
+        "CoreAlertFields"
```

Comment:
Why did you add this change?

Reply:
It was generated automatically by demisto-sdk commands. You can remove it if you want (I don't understand the meaning of it).